packages/hash/frontend/src/pages/signup.page/account-usage.tsx

```diff
@@ -15,13 +16,13 @@ const USAGE_OPTIONS = [
     icon: <HumanGreetingIcon className={tw`mb-6 w-20 h-20`} />,
     title: "By myself",
     description: "Create a personal knowledge graph",
-    value: WayToUseHash.ByThemselves,
+    value: "BY_THEMSELVES",
```

Comment:
Quite inline with the conversation that was happening in Slack earlier today about enums 🙃 Really don't like these magic strings all over the place.. (not asking for a change)

Reply:
> magic strings all over the place

😆